docs/earthfile/earthfile.md

```diff
@@ -1598,12 +1595,37 @@ The `PIPELINE` command is in beta status and is only useful for Earthly CI.
 
 #### Description
 
-The `PIPELINE` command is used to declare that the current target is an Earthly CI pipeline. The `PIPELINE` command must be the first command in the target.
+The `PIPELINE` command is used to declare that the current target is an Earthly CI pipeline. The `PIPELINE` command must be the first command in the target. The `PROJECT` declaration in the Earthfile is also required, and it must match the project the repository is added to, in order for the pipeline to be executed.
 
 A pipeline is a target that is executed by Earthly CI when a certain trigger is activated. Triggers can be declared via the `TRIGGER` command. Pipeline targets allow only the commands `TRIGGER`, `ARG` and `BUILD`. Other commands may be used indirectly in other targets that can be then referenced by `BUILD`.
 
 Pipeline targets are always executed with no outputs, in strict mode.
 
+{% hint style='info' %}
+##### Note
+Pipelines and their definitions, including their triggers must be merged into the primary branch (by default, the default branch on GitHub) in order for the triggers to take effect.
```

Comment:
`by default, the default` is difficult to quickly parse, maybe we can use `overridden` for one of the `default`s instead?
```suggestion
Pipelines and their definitions, including their triggers must be merged into the primary branch (which, unless overridden, is the default branch on GitHub) in order for the triggers to take effect.
```